middleware/upload.js

```diff
@@ -0,0 +1,17 @@
+const multer = require("multer");
+const path = require("path");
+const uploadDir = path.join(process.cwd(), "tmp");
```

Comment:
Ponieważ używasz tej ścieżki w kilku miejscach, to dobrym pomysłem byłoby wyniesienie jej gdzieś indziej do jakiegoś pliku np common.js i exportowanie jej tam. Będziesz mieć pewność że jak zmienisz w jednym miejscu to wszędzie nastąpi zmiana.